main.go

```diff
@@ -52,7 +52,8 @@ func newMessage(cfg config, buildSuccessful bool) Message {
 	// MessageCard sections
 	primarySection := buildPrimarySection(cfg)
 	factsSection := buildFactsSection(cfg, buildSuccessful)
-	message.Sections = []Section{primarySection, factsSection}
+	imagesSection := buildImagesSection(cfg)
+	message.Sections = []Section{primarySection, factsSection, imagesSection}
```

Comment:
We should probably swap the positions of `factsSection` and `imagesSection` in the `Section` slice, as the slice index determines the render positions within the MessageCard. The Fact section should typically be the bottom-most section before the action buttons within the card.

Reply:
Sounds good to me.